docs/common-coreboot-docs/dasharo_tools_suite.md

```diff
@@ -2,30 +2,89 @@
 
 ## Introduction
 
-Dasharo Tools Suite (DTS) is a set of tools for deploying, updating and
-maintaining firmware on Dasharo supported devices. DTS is integrated into the
-firmware in Dasharo supported devices and can be used to update the firmware
-even when no OS is installed.
+Dasharo Tools Suite (DTS) is a set of tools running in a minimal Linux
+environment, with a goal of deploying, updating and maintaining firmware on
+Dasharo supported devices. For example, it can be used to update the firmware
+on a device, even when no OS is currently installed.
 
-## Requirements
+## Distribution methods
 
-- Dasharo device with DTS functionality integrated
-- Wired network connection
-- Secure Boot disabled
+It can be distributed in various ways. Currently, there are two distribution
+options:
 
-## Launching DTS
+* bootable USB stick image.
+* bootable over network (iPXE),
+
+## Flavors
+
+There is a common base, but there might be multiple flavors of the DTS images.
+Currently, there are:
+
+* CE
+    - `Dasharo HCL Report` - generates a package with logs containing hardware
+      information
+    - flashrom, fwupd, and many more useful tools
+    - can drop to shell to update the firmware manually
+* OEM
+    - on top of that, provides tools for automatic firmware deployment and
+      rollback (switching to Dasharo back and forth)
+
+## Bootable USB stick
+
+This documentation is compatible with the `v0.3.0` version of the DTS.
```

Comment:
This sounds like a problem. It would mean we need a versioning for the documentation. Now:
1. Does it mean the documentation for newer versions will not be backwards compatible?
2. How to check the documentation for older version in the future, e.g. when we release v1.2.3 for example, how to check v1.1.1 if someone still uses it?

We could maybe assume that the documentation is extend-only where each extension is marked with the first version since feature introduction. I.e. supported since version vx.y.z?

Reply:
> Does it mean the documentation for newer versions will not be backwards compatible?

I don't know maybe. I can easily image that yes. Everything can change here at this point.